modules/nginx/src/test/java/org/testcontainers/junit/SimpleNginxTest.java

```diff
@@ -20,18 +17,14 @@
  */
 public class SimpleNginxTest {
 
-    @Rule
-    public NginxContainer nginx = new NginxContainer()
-            .withCustomContent(System.getProperty("user.home") + "/.tmp-test-container");
+    private static File contentFolder = new File(System.getProperty("user.home") + "/.tmp-test-container");
```

Comment:
For sure, but I don't want to touch this classes too much during this PR because it's unrelated :)